builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -509,9 +514,31 @@ func (b *backend) pathGetRawIssuer(ctx context.Context, req *logical.Request, da
 		return nil, err
 	}
 
-	certificate := []byte(issuer.Certificate)
-
 	var contentType string
+	var certificate []byte
+
+	// tmp
+	responseHeaders := map[string][]string{}
+	if hasHeader(headerIfModifiedSince, req) {
+		before, err := sc.isIfModifiedSinceBeforeIssuerLastModified, ref, responseHeaders)
+		if err != nil {
+			return nil, err // errorResponse, nil or nil, err?
```

Comment:
Its an internal error so I'm fine with `return nil, err`. 

I _think_ `errorResponse` is mostly for bad user inputs. @stevendpclark ?

Reply:
Yes that is my understanding.